spec/controllers/socure_webhook_controller_spec.rb

```diff
@@ -6,12 +6,16 @@
   describe 'POST /api/webhooks/socure/event' do
     let(:socure_secret_key) { 'this-is-a-secret' }
     let(:socure_secret_key_queue) { ['this-is-an-old-secret', 'this-is-an-older-secret'] }
+    let(:socure_webhook_enabled) { true }
 
     before do
       allow(IdentityConfig.store).to receive(:socure_webhook_secret_key).
         and_return(socure_secret_key)
       allow(IdentityConfig.store).to receive(:socure_webhook_secret_key_queue).
         and_return(socure_secret_key_queue)
+      allow(IdentityConfig.store).to receive(:socure_webhook_enabled).
+        and_return(socure_webhook_enabled)
+      Rails.application.reload_routes!
```

Comment:
Another upside to handling in the controller is we won't need to reload routes like this.

```suggestion
```